debezium-embedded/src/main/java/io/debezium/embedded/AsyncEmbeddedEngine.java

```diff
@@ -0,0 +1,1266 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.embedded;
```

Comment:
Maybe it would be better to move this class to a separate package `io.debezium.embedded.async` and split it file into multiple classes?

Reply:
ack (actually I intended to do so before marking ready for review ... obviously I forgot:-()

Reply:
done now